README.md

```diff
@@ -194,63 +206,80 @@ Now you can run the billing_alert.cfn.yml template, which will create a [CloudWa
 You can read about more [ways to avoid unexpected charges](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/checklistforunwantedcharges.html).
 
 
+### Launching Modular Stacks
 
-<a name="launch-table"></a>
-### Launch stack
-
-Click a row's "Launch stack" button to launch a single stack in the specified region containing all the resources in the checked templates.
+Select the Category of stack you want to launch below. Then find the row with the combination of modules you are looking for from the checkbox columns (i.e. vpc+bastion host) and select the region you want to launch the stack in. Click 'Launch Stack' button and the CloudFormation console will open automatically with the stack's details.
 
 New services are not immediately available in all AWS Regions, please consult the [Region Table](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) for more information.
 
```

Comment:
I like hiding these large tables under a toggle, but it could be easy to miss them entirely. Maybe they should be preceded by an explainer, something like "Click a heading below to launch a CloudFormation stack in the region of your choice," perhaps.

Reply:
I agree and had the same comment above. Please add in next iteration.